tests/cypress/e2e/errorDisplay.cy.ts

```diff
@@ -0,0 +1,42 @@
+import {JContent} from "../page-object";
+
+describe('Validate error handling', () => {
+
+    const sitekey = 'jcontentSiteError';
+    before(() => {
+        cy.executeGroovy('jcontent/createSite.groovy', {SITEKEY: sitekey});
+    })
+
+    after(() => {
+        cy.logout();
+        cy.executeGroovy('jcontent/deleteSite.groovy', {SITEKEY: sitekey});
+    });
+
+    beforeEach(() => {
+        cy.loginEditor();
+        JContent.visit(sitekey, 'en', 'pages/home').switchToPageComposer();
```

Comment:
do we really need this ? each test is doing another "visit" command at the beginning

Reply:
want to be sure, all else is working before checking for errors

Reply:
moving it to just before() to check only once